lib/routes/account.js

```diff
@@ -536,6 +536,11 @@ module.exports = function (
           response.keyFetchToken = keyFetchToken.data.toString('hex')
 
           if (doSigninConfirmation) {
+            log.activityEvent('account.signin.verify.start', request, {
```

Comment:
Should we also log one of these during `/account/create` in order for the total `verify.start` to equal the theoretical total `verify.success` events?